schema/schema.go

```diff
@@ -13,6 +13,20 @@ import (
 	"gorm.io/gorm/logger"
 )
 
+type callbackType string
+
+const (
+	_callbackTypeBeforeCreate callbackType = "BeforeCreate"
+	_callbackTypeBeforeUpdate callbackType = "BeforeUpdate"
+	_callbackTypeAfterCreate  callbackType = "AfterCreate"
+	_callbackTypeAfterUpdate  callbackType = "AfterUpdate"
+	_callbackTypeBeforeSave   callbackType = "BeforeSave"
+	_callbackTypeAfterSave    callbackType = "AfterSave"
+	_callbackTypeBeforeDelete callbackType = "BeforeDelete"
+	_callbakAfterDelete       callbackType = "AfterDelete"
```

Comment:
Here is a typo, could you also use remove all _ from names? it is not exported at the package level and can't conflict with anything since it has its own type.

Reply:
There is a spelling mistake; it should be '_callbackAfterDelete' instead of '_callbakAfterDelete'.